pages/dkp/kommander/2.1/projects/applications/catalog-applications/dkp-applications/custom-resources-workspace-catalog/zookeeper/index.md

```diff
@@ -16,6 +16,8 @@ A [Helm chart](https://github.com/pravega/zookeeper-operator/tree/master/charts/
 
 ### Example deployment
 
+<p class="message--note"><strong>NOTE: </strong>The Custom Resources need to be applied to each attached cluster in a project.</p>
```

Comment:
```suggestion
<p class="message--note"><strong>NOTE: </strong>The Custom Resources must be applied to each attached cluster in a project.</p>
```

Same comment here re tweaking paragraph above

Reply:
fixed in d58e0411b0e74dbe4797a6be8f8cc146c1c1019e!